CHANGES.md

```diff
@@ -1,3 +1,29 @@
+## 1.5.1 2018-10-24
+
+Linkerd 1.5.1 adds an exciting new interpreter to its suite of interpreter module. The new 
+`io.l5d.consul.interpreter` now allows Linkerd to read dtabs directly from a Consul KV store agent
+that This effectively removes the dependence on Namerd to resolve service names to concrete names
+and simplifies Linkerd's deployment model in Consul backed architectures. In addition, this release
+adds a number of critical bug fixes to the HTTP/2 router that fixes an issue where Linkerd would 
+sometimes degrade HTTP/2 streaming over some period of time.
```

Comment:
"degrade HTTP/2 streaming" is vague and I don't really know what it means.  Should probably be more specific (or less specific and just let the changelist describe it in more detail)